active/TreeView/TreeViewDragItemsCompletedNewParent.md

```diff
@@ -0,0 +1,43 @@
+# Background
+
+TreeView has built-in support for drag&drop reorder but currently there is no easy way to figure out where the item is being dropped. 
+
+# Description
+
+TreeView's data usually has hierarchical structure by nature and "index" doesn't work very well in this scenario. Parent + SubIndex is a more accurate way to locate items in TreeView. 
+
+A `NewParent` property in TreeViewDragItemsCompletedEventArgs should be sufficient to represent the drop location. To get SubIndex, developers can do NewParent.Children.IndexOf().
+
+# Examples

```

Comment:
This example isn't very informative, especially with NewParent being un-typed. Could you add some detail? Suggestion: assume the TreeView is showing a Dictionary<string,List<string>>, where the first string is team names and the second string is player names. Then in the event handler set a TextBlock.Text to "'{playerName}' has changed to the {teamName}".

Reply:
Sure I'll update : )